addon/hint/show-hint.js

```diff
@@ -293,6 +293,7 @@
       }
     }
     var overlapX = box.right - winW;
+    if (scrolls) overlapX += cm.display.nativeBarWidth;
```

Comment:
as mentioned, the amount of padding used in the hint items when there is a scrollbar is also added to the overlapX, moving the hints container just a smidge to the left, so the scrollbar is not off screen...

Reply:
Much better, thanks!

Reply:
thank YOU for building such awesome tools! :)